configure.py

```diff
@@ -1256,6 +1284,29 @@ def configure_library(lib, output, pkgname=None):
     elif pkg_libs:
       output['libraries'] += pkg_libs.split()
 
+def configure_static_library(lib, output):
+  static_lib = 'static_' + lib
+  output['variables']['node_' + static_lib] = b(getattr(options, static_lib))
+
+  if getattr(options, static_lib):
+    # Apply to all Node.js components for now
+    if options.__dict__[static_lib + '_includes']:
+      output['include_dirs'] += [options.__dict__[static_lib + '_includes']]
+
+    # Set library path
+    if options.__dict__[static_lib + '_libpath']:
+        output['libraries'] += [
+            '-L%s' % options.__dict__[static_lib + '_libpath']]
+
+    default_libs = getattr(options, static_lib + '_libname')
+    default_libs = ['-l{0}'.format(l) for l in default_libs.split(',')]
+
+    if default_libs:
+      output['libraries'] += default_libs
+
+def configure_zos(output):
+  # configure zoslib library only for now on z/OS
```

Comment:
Not sure if this comment is necessary, might be confusing as a temporary change.